<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1021</width>
    <height>739</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="CanvasView" name="canvasView" native="true">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>611</width>
      <height>611</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="colorPickerButton">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>50</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="FrameListView" name="frameListView" native="true">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>120</width>
      <height>611</height>
     </rect>
    </property>
   </widget>
   <widget class="ToolbarView" name="toolbarView" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>101</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="colorPickerLabel">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>30</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Color Picker</string>
    </property>
   </widget>
   <widget class="QLabel" name="brushSizeLabel">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>130</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Brush Size</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="brushSizeSpinBox">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>150</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>118</width>
      <height>213</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="resizeButton">
         <property name="text">
          <string>Resize</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="importPNGButton">
         <property name="text">
          <string>Import PNG</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="exportPNGButton">
         <property name="text">
          <string>Export PNG</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="saveButton">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="loadButton">
         <property name="text">
          <string>Load</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="showPreviewButton">
         <property name="text">
          <string>Show Preview</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="cursorPositionLabel">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>620</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>X: 0 Y: 0</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1021</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CanvasView</class>
   <extends>QWidget</extends>
   <header>canvasview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FrameListView</class>
   <extends>QWidget</extends>
   <header>framelistview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ToolbarView</class>
   <extends>QWidget</extends>
   <header>toolbarview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
